pom.xml

```diff
@@ -177,6 +177,25 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
         <version>2.3.1</version>
+        <configuration>
+          <archive>
+            <manifest>
+              <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+              <addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+            </manifest>
+            <manifestEntries>
+              <Bundle-Version>${parsedVersion.osgiVersion}</Bundle-Version>
+              <Bundle-Description>${project.description}</Bundle-Description>
+              <Os-Name>${os.name}</Os-Name>
+              <Os-Arch>${os.arch}</Os-Arch>
+              <Os-Version>${os.version}</Os-Version>
+              <X-Compile-Source-JDK>${maven.compiler.source}</X-Compile-Source-JDK>
```

Comment:
Apache does it for their commons modules.  I've seen it used in lots of other places over the years.  I don't know if it is for any specific standard.  It has nothing to do with OSGi.  This from what I understand makes it easy for me as a user of a library to know if it will work against a given JVM.  Without it, the build just tells us the product was built with some version of java.  If that version of java is higher than we use, one questions if it can be used.  This clears up that confusion.  I have no idea if this assists for anything otherwise other than I had read long ago it was good practice.